docs/ZendeskMultiObjects-batchsource.md

```diff
@@ -3,20 +3,14 @@
 
 Description
 -----------
-This source reads multiple objects from Zendesk. It extracts reportable data from the multiple Zendesk objects. 
-The Zendesk Multi Object Batch Source plugin enables bulk data extraction from Zendesk. 
-You can configure and execute bulk data transfers from Zendesk without any coding.
+The Zendesk Multi Object Batch Source plugin provides seamless extraction of reportable data from multiple objects 
+within Zendesk.
 
-Examples of objects are Article Comments, Post Comments, Requests Comments, Ticket Comments,
-Groups, Organizations, Satisfaction Ratings, Tags, Ticket Fields,
-Ticket Metrics, Ticket Metric Events, Tickets, Users.
-Tags object lists the 500 most popular tags in the last 60 days.
-
-The data which should be read is specified using object list and filters for those objects.
-
-In addition, for each Object that will be read, this plugin will set pipeline arguments where the key is 
+In addition, for each Object that's read, this plugin will set pipeline arguments where the key is 
 'multisink.[ObjectName]' and the value is the schema of the Object.
```

Comment:
Nit: I think 'multisink.[ObjectName]' should instead be `multisink.[ObjectName]` (` is markdown for code font).